roles/install_ansible_rulebook/tasks/install_macosx.yml

```diff
@@ -18,42 +21,49 @@
     state: link
   become: true
 
-- shell: /usr/libexec/java_home
+- name: Register java_home
+  ansible.builtin.command: /usr/libexec/java_home
   register: java_home
+  changed_when: java_home.rc != 0
+  failed_when: java_home.rc != 0
 
 - name: set user envvars
```

Comment:
Can you trigger a test run? That task name has been updated, and I don't know why it checks old code.
```sh
$ git grep envvars roles/
roles/install_ansible_rulebook/tasks/install_fedora.yml:- name: Set user envvars
roles/install_ansible_rulebook/tasks/install_macosx.yml:- name: Set user envvars
```

Reply:
Just need to add a couple of newlines and it should be good to go.
https://github.com/ansible/event-driven-ansible/actions/runs/3381227200/jobs/5621457845

Reply:
yeah, but the annoying part is that it isn't triggered locally...
```sh
$ ansible-lint roles/install_ansible_rulebook/

Passed with production profile: 0 failure(s), 0 warning(s) on 11 files.
```